### PR TITLE
chore: changed package, context path, port etc

### PR DESCRIPTION
**Pull request by jayanta2018**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,2 +1 @@
-// TODO: Update project name.
-rootProject.name = 'template'
+rootProject.name = 'migration'
```
Comment (DorisWongNHS) [inline settings.gradle:1]: Wondering should it be `tis-revalidation-migration`

File: build.gradle [hunk 2]
```diff
@@ -9,8 +9,7 @@ plugins {
   id "org.sonarqube" version "3.2.0"
 }
 
-// TODO: Update group to end with "admin" or "trainee".
-group = "uk.nhs.hee.tis"
+group = "uk.nhs.hee.tis.revalidation.migration"
```
Comment (DorisWongNHS) [inline build.gradle:12]: Should the group = "uk.nhs.hee.tis.revalidation" ?

**Review by CaiWillis — COMMENTED** [review]
Do we want to change the names in the .aws folder task definitions too at this point before merging? or can that wait until you do the full work on the TDs?

**Review by DorisWongNHS — CHANGES_REQUESTED** [review]

**Comment by jayanta2018** [comment]
> Do we want to change the names in the .aws folder task definitions too at this point before merging? or can that wait until you do the full work on the TDs?

May be we can wait until we start working on TD.

File: build.gradle (hunk 2 above)
Comment (jayanta2018) [reply build.gradle]: Done

File: settings.gradle (hunk 1 above)
Comment (jayanta2018) [reply settings.gradle]: Done

**Review by DorisWongNHS — APPROVED** [review]

**Pull request merged**